packages/block-editor/tsconfig.json

```diff
@@ -4,10 +4,14 @@
 		"rootDir": "src",
 		"declarationDir": "build-types"
 	},
+	"references": [
+		{ "path": "../element" }
+	],
```

Comment:
This actually depends on a bunch of things:
https://github.com/WordPress/gutenberg/blob/e00a4850737c89860d98f5830cf2b0b09bc0d210/packages/block-editor/package.json#L26-L48

We should add the packages that are typed already at least.

Reply:
> This actually depends on a bunch of things:

That package as a whole does, sure. Currently we only type-check two files, for which this `references` set is accurate. I guess it can save us some future work if we just go ahead and add all of the dependencies now?

Reply:
This is interesting to consider…

I've been thinking of the TypeScript project as corresponding to the JavaScript project, but in this case it really doesn't. It's a small subset of the block-editor package. As such, it has a subset of the dependencies.

I don't feel strongly about this one way or the other.